sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/TransmissionStateManager.cs

```diff
@@ -0,0 +1,94 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
+{
+    internal class TransmissionStateManager
+    {
+        private int _consecutiveErrors;
+
+        private const long MaxDelayInMilliseconds = 3600000;
+
+        internal const long MinDelayInMilliseconds = 10000;
+
+        private readonly Random _random = new();
+
+        private readonly TimeSpan _minIntervalToUpdateConsecutiveErrors = TimeSpan.FromMilliseconds(20000);
+
+        private DateTimeOffset _nextMinTimeToUpdateConsecutiveErrors = DateTimeOffset.MinValue;
```

Comment:
nit: Consider leaving a comment, if not we have to refer to `DateTimeOffset.MinValue` doc.